app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/ImageRegionSelectionInteractionViewModel.kt

```diff
@@ -42,6 +44,7 @@ class ImageRegionSelectionInteractionViewModel private constructor(
     schemaObject?.customSchemaValue?.imageWithRegions?.imagePath ?: ""
   }
 
+  val clickedRegions = rawUserAnswer?.imageRegionSelection ?: ClickOnImage.getDefaultInstance()
```

Comment:
Here, I am retaining the selected image region but it is not showing up in UI. I have also update it's corresponding xml file and `ImageRegionSelectionInteractionView` and added new properlty but still it's not working.